build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/__component_version__/amp-__component_name_hyphenated__.js

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import {CSS} from './__component_name_hyphenated__.jss';
```

Comment:
Should we remove the `.css` file?

Reply:
AFAIK the CSS is still usable

Reply:
Will leave it in for now unless someone clarifies that it's explicitly disallowed

Reply:
It's ok to keep, some components do have a separate css file.